### PR TITLE
JavaScript Project

### PR DESCRIPTION
**Pull request by luqmanmusah**
The JavaScript capstone project is about building your own web application based on an external API. We selected an API that provides data about a topic that we like and then builds the webapp around it. 

We worked on an API showing movies.

A home page showing a list of items that you can "like."
A popup window with more data about an item that you can use to comment on it or reserve it for a period of time.

Here is a short [video demo](https://drive.google.com/file/d/1KVvZPCVzUw1rDxo7jHRvn836m3Co1C1Y/view?usp=sharing)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <li>
+          <a href="#">second link</a>
+        </li>
+        <li>
+          <a href="#">third link</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <ul class="movies" id="movies">
```
Comment (vichuge) [inline index.html:28]: - [x] For each movie is important to use an id, this helps to have a better order for elements. Please add the id for each element on this list. 🐶
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  </ul>
+  <!-- Full screen modal -->
+<div class="modal" id="myModal">
+   <!-- Modal content -->
+   <div class="modal-content" id="modalContent">
+  </div>
+</div>
+
+  <!-- <footer>
```
Comment (vichuge) [inline index.html:38]: - [x] It looks like the page doesn't contain a footer, please add it. A footer is important on a page as close the content and for SEO purposes. 🐱

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    </form>
+  `;
+
+  const commentUl = document.getElementById('commentUl');
+  let commentList = await getComments();
+  if (commentList.error) {
+    commentList = [];
+  }
+
+  commentList.forEach((comment) => {
```
Comment (vichuge) [inline src/index.js]: - [x] When I add a new comment, the modal close automatically and then I need to enter again on the movie modal to see my comment, please avoid this step, when I add a comment, the modal movie needs to stay there and appear the new comment on the list without close the modal. Please, work on this 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] It looks like there's not a video on your project, I kindly invite you to add the video presentation following this [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=Once%20you%20have,with%20your%20peers.)

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by m05e5** [comment]
Thanks for the review. I implemented the changes you suggested.

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Updating the README live demo link to reflect the recent changes 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**